tls/s2n_ktls_io.c

```diff
@@ -273,7 +273,7 @@ S2N_RESULT s2n_ktls_recvmsg(void *io_context, uint8_t *record_type, uint8_t *buf
  * We can estimate the number of "full-sized records" sent by assuming that
  * all records are full-sized.
  */
-static S2N_RESULT s2n_ktls_estimate_records(size_t bytes, uint64_t *estimate)
+S2N_RESULT s2n_ktls_estimate_records(size_t bytes, uint64_t *estimate)
```

Comment:
Nit: maybe we should move all these helper methods (s2n_ktls_estimate_records, s2n_ktls_check_estimated_record_limit, and s2n_ktls_set_estimated_sequence_number) into s2n_ktls.c. Then s2n_ktls_estimate_records can stay static, and s2n_ktls_check_estimated_record_limit and s2n_ktls_set_estimated_sequence_number would go in s2n_ktls.h.